Algebraic_kernel_d/examples/Algebraic_kernel_d/CMakeLists.txt

```diff
@@ -18,20 +18,17 @@ endif()
 include( ${CGAL_USE_FILE} )
 
 
-find_package( Boost REQUIRED )
+# Boost and its components
+find_package( Boost QUIET )
 if ( NOT Boost_FOUND )
   message(STATUS "This project requires the Boost library, and will not be compiled.")
-  return()  
+  return()
 endif()
 
-
-find_package( MPFI REQUIRED )
-if ( NOT MPFI_FOUND )
-  message(STATUS "This project requires the MPFI library, and will not be compiled.")
-  return()  
+find_package( MPFI QUIET )
```

Comment:
Actually, nobody in the CGAL project now understands what should happen when non-CGAL components are listed in the `find_package(CGAL .. )` command. It was a feature added in our CMake scripts by Eric and Philipp, and they both left, without documenting that feature correctly.

It was the "small" feature [`Features/Small_Features/CMake-Installation-Manual`][sf] (link for CGAL developers only).

[sf]: https://cgal.geometryfactory.com/CGAL/Members/wiki/Features/Small_Features/CMake-Installation-Manual

I plan to remove the CMake code about those pre-configured third party libraries pretty soon (next week).